app/views/admin/locations/_confirm_delete_location.html.haml

```diff
@@ -19,4 +19,4 @@
         permanently.
     .modal-footer
       %button.btn.btn-default{ 'data-dismiss' => 'modal' } Close
```

Comment:
We should also move that to a shared partial as the same code is used on other delete views.